Services/ImportMultimediaObjectService.php

```diff
@@ -258,12 +210,19 @@ public function setMultimediaObject($mmArray, $series)
             }
         }
 
-
         $multimediaObject->setStatus(intval($multimediaObject->getStatus()));
 
         $multimediaObject->setRecordDate(new \Datetime($multimediaObject->getRecordDate()));
         $multimediaObject->setPublicDate(new \Datetime($multimediaObject->getPublicDate()));
 
+        if ($multimediaObject->isOnlyAudio()) {
+            $multimediaObject->setType(MultimediaObject::TYPE_AUDIO);
```

Comment:
These changes only works with PuMuKIT 2.4. We should create a new branch `2.0.x`.